documentation/man/features/report.rst

```diff
@@ -39,13 +55,15 @@ OPTIONS
 \--year[=<year>]:
   Exhibits the current year summary if the user does not specify *<year>*.
 
+\--statistics:
+  Display current date statistics summary.
```

Comment:
Change this to be more generic and take into account the date parameters. Something like this:
`Display statistics for the given date.`
You could alternatively just add something like `You can choose the date and range with the date options.`